.Rbuildignore

```diff
@@ -2,17 +2,13 @@
 ^.*\.Rproj$
 ^\.Rproj\.user$
 ^LICENSE\.md$
-^working$
 ^R/_render.R$
-^Untitled$
 ^README.Rmd$
```

Comment:
Leave these in please as they are on my system locally and they need to be ignored when the package is built and submitted.